flytekit/core/task.py

```diff
@@ -111,10 +111,16 @@ def my_task(x: int, y: typing.Dict[str, str]) -> str:
            but you can always manually update this field as well to force a cache miss. You should also manually bump
            this version if the function body/business logic has changed, but the signature hasn't.
     :param retries: Number of times to retry this task during a workflow execution.
-    :param interruptable: Boolean that indicates that this task can be interrupted and/or scheduled on nodes
+    :param interruptible: [Optional] Boolean that indicates that this task can be interrupted and/or scheduled on nodes
```

Comment:
@katrogan @wild-endeavor: Does this explanation make sense?

Reply:
I think this sounds right, but I'm wondering what happens when workflow has interruptible=True and task has nothing set. Presumably the task should then inherit the workflow value, but we always default the task interruptible param value to False. So does this always get overwritten?

Reply:
hey @jeevb I'm sorry to pile on more work, but according to the original interruptible change, we defaulted the task interruptible bit to [None, rather than False](https://github.com/flyteorg/flytekit/pull/86/files#diff-4705dca3eaeaae1ac85b8f20caede8a7e8d6561da90c3fb5c7d8d6713df7e0a5R117) - would you mind changing that in [core/task.py](https://github.com/flyteorg/flytekit/blob/master/flytekit/core/task.py#L67)? I think that's all that's required to make sure we don't set the oneof bit for Task/Node metadata at serialization time

cc @migueltol22

Reply:
The task default has been changed to `None`. So the task should inherit from the workflow. I'll add this case to the readme.

Reply:
oh already done, nvm!!